autogen.sh

```diff
@@ -90,12 +90,19 @@ fi
 #    grep -i '">' tools/nut-usbinfo.pl
 # * List the names involved:
 #    grep -E 'output.*=' tools/nut-usbinfo.pl
+# Also check that the last re-generation is newer than the sources involved
+# (stay on top of CI rebuilds, development, Git branch switching...)
+# Someone please tell me why GNU `find dir -newer X -name Y -o -name Z` does
+# not filter away layer by layer, but rather finds the names Z and beyond
```

Comment:
I think that means newer than X and Y, or called Z (with no newer).  And binds tighter.  You need () maybe.   But what you have seems sensible and I don't suggest you redo it.

Reply:
I guessed so, but no sort of explicit `-and`/`-or` magic helped. Not sure how portable parentheses are, so went for plain `-newer` as something all find's I've seen can do  :)